.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Request reviews from the iOS app maintainer by default
+* @beeminder/ios-maintainer
```

Comment:
this team has to be created elsewhere, which I am guessing you did


Is it named "a team"? Maybe Beeminder Maintainer or BeeSwift Maintainers could be the name?

![Screenshot 2024-12-29 at 17 50 19](https://github.com/user-attachments/assets/f54f8ca8-0b1e-407c-a24f-ec4d2a7eecdd)

Reply:
It's called `ios-maintainer`. Unfortunately github doesn't support making it public: https://github.com/orgs/community/discussions/20808

Reply:
This is how those lines show up for me.

<img width="813" alt="image" src="https://github.com/user-attachments/assets/d087cff8-8d77-427c-8f4d-308e6b3b8b13" />